aes-siv/tests/vectors/aead.rs

```diff
@@ -0,0 +1,86 @@
+#![allow(dead_code)]
+
+pub use serde_json::Value as JsonValue;
+use subtle_encoding::hex;
+
+/// AES-SIV AEAD test vectors
+// TODO(tarcieri): extract these from JSON format
```

Comment:
I would like to extract these vectors from the JSON before merging (using e.g. `hex-literal` instead)